internal/server/rest_test.go

```diff
@@ -1004,3 +1006,29 @@ func (suite *RestTestSuite) TestWaitStopRule() {
 	require.True(suite.T(), end.Sub(now) >= 300*time.Millisecond)
 	waitAllRuleStop()
 }
+
```

Comment:
Suggest not to add new case to this file any more because it needs a lot of duplicate code to mock. Just add a test case in fvt